src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DEFAULT_DELIMITER = ",|:";
+    private static final String CUSTOM_DELIMITER_PATTERN = "//(.)\n(.*)";
+
+    public static int splitAndSum(String input) {
+        if (isEmptyString(input)) {
+            return 0;
+        }
+
+        return sum(split(input));
+    }
+
+    private static boolean isEmptyString(String str) {
+        return str == null || str.length() == 0;
```

Comment:
```suggestion
        return str == null || str.isEmpty();
```
같은 동작이라면 많은 사람들이 사용함으로써 검증된 기본 API를 사용하는 것도 좋은 방식인 것 같습니다 😄